packages/server/tests/AuthenticationSignUp.test.ts

```diff
@@ -62,10 +63,15 @@ test.group('Authentication sign up tests group', (group) => {
                 address: 'no-reply@adonisenprovence.com',
             });
 
-            assert.equal(
-                message.subject,
-                `Welcome ${userNickname}, please verify your email !`,
+            const emailVerificationObjectRegex = new RegExp(
+                `[.*].*Welcome.*${userNickname},.*please.*verify.*your.*email.*!`,
             );
+            assert.isDefined(message.subject);
+            invariant(
+                message.subject !== undefined,
+                'message subject is undefined',
+            );
+            assert.isTrue(emailVerificationObjectRegex.test(message.subject));
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** You can use `assert.match()` to test a string against a Regex. It does the same thing as here. 😄

Reply:
About to be fixed